main.tf

```diff
@@ -70,6 +68,12 @@ resource "aws_s3_bucket" "website" {
   }
 
   tags = local.tags
+
+  lifecycle {
+    ignore_changes = [
+      lifecycle_rule
```

Comment:
does acs modify the lifecycle_rule?

Reply:
Yes. It adds a rule to abort multipart uploads.